src/MOI_wrapper.jl

```diff
@@ -347,12 +347,12 @@ end
 
 function MOI.get(model::Optimizer,
     ::ForwardIn{QuadraticObjective}, vi1::VI, vi2::VI)
-    tuple = ifelse(index(vi1) <= index(vi2), (vi1, vi2), (vi2, vi1))
+    tuple = ifelse(vi1.value <= vi2.value, (vi1, vi2), (vi2, vi1))
```

Comment:
why changing this one?

Reply:
it was failing on my device, on Julia 1.0

Reply:
![image](https://user-images.githubusercontent.com/20796893/116872725-03629000-ac34-11eb-809d-e3a7e8b629e7.png)

Reply:
one thing to note: `tuple` is also a Julia function in Base, maybe try to use a name that is not already defined to avoid tricky bugs